packages/fxa-payments-server/src/images/icon-check-white.svg

```diff
@@ -0,0 +1,3 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" viewBox="0 0 16 16">
```

Comment:
Should this go in [`public/images`](https://github.com/mozilla/fxa/tree/master/packages/fxa-payments-server/public/images) instead? Or does this get picked up by some webpack processing as a dependency?

Reply:
Good catch. Updated and verified locally.

Reply:
I think this is a weird artifact of our efforts to reuse style sheets across packages that both use different build processes 😞